medical_his/views/medical_hospital_bed_view.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="medical_hospital_bed_search_view" model="ir.ui.view">
+            <field name="name">medical.hospital.bed.search</field>
+            <field name="model">medical.hospital.bed</field>
+            <field name="arch" type="xml">
+                <search string="Medical Hospital Bed" version="7.0">
+                    <field name="name" />
+                    <field name="display_name" />
+                    <field name="state" />
+                    <field name="phone" />
+                    <field name="notes" />
+                    <field name="room_id" />
+
+                    <filter name="free" string="Free" domain="[('state','=', 'free')]" />
+                    <separator />
```

Comment:
I think you don't want this separator here.
This will make you search if you click on free and occupied

    state == 'free' and state == 'occupied'

And my guess is that you wont get much results :wink: